package.json

```diff
@@ -131,6 +131,7 @@
     "@vue/test-utils": "^2.4.6",
     "@vue/tsconfig": "^0.5.1",
     "babel-loader-exclude-node-modules-except": "^1.2.1",
+    "core-js": "^3.39.0",
```

Comment:
A part of backport?

Reply:
yes we have this on master, but missing here (so we "implicitly" depending on it, thats why it works, but we directly import from it for tests / docs so we should explicitly mark it in the dev dependencies).

Yes I can split this off, but its one line :see_no_evil: